Bio/SubsMat/MatrixInfo.py

```diff
@@ -15,17 +15,48 @@
 
 
 # a list of all available substitution matrices
-available_matrices = ["benner6", "benner22", "benner74", "blosum100",
-                      "blosum30", "blosum35", "blosum40", "blosum45",
-                      "blosum50", "blosum55", "blosum60", "blosum62",
-                      "blosum65", "blosum70", "blosum75", "blosum80",
-                      "blosum85", "blosum90", "blosum95", "feng",
-                      "fitch", "genetic", "gonnet", "grant",
-                      "ident", "johnson", "levin", "mclach",
-                      "miyata", "nwsgappep", "pam120", "pam180",
-                      "pam250", "pam30", "pam300", "pam60",
-                      "pam90", "rao", "risler", "structure",
-                      ]
+available_matrices = [
+    "benner6",
+    "benner22",
+    "benner74",
+    "blosum100",
+    "blosum30",
+    "blosum35",
+    "blosum40",
+    "blosum45",
+    "blosum50",
+    "blosum55",
+    "blosum60",
+    "blosum62",
+    "blosum65",
+    "blosum70",
+    "blosum75",
+    "blosum80",
+    "blosum85",
+    "blosum90",
+    "blosum95",
+    "feng",
+    "fitch",
+    "genetic",
+    "gonnet",
+    "grant",
+    "ident",
+    "johnson",
+    "levin",
+    "mclach",
+    "miyata",
+    "nwsgappep",
+    "pam120",
+    "pam180",
+    "pam250",
+    "pam30",
+    "pam300",
+    "pam60",
+    "pam90",
+    "rao",
+    "risler",
+    "structure",
+]
```

Comment:
The black version of this list is very long - this could be an example where locally disabling the black formatting, and instead having manual line wrapping at 88 characters could be preferable. What do you think?